crates/bevy_ui/src/widget/mod.rs

```diff
@@ -5,7 +5,21 @@ mod image;
 mod slider;
 mod text;
 
+use bevy_app::{Plugin, App};
 pub use button::*;
 pub use image::*;
 pub use slider::*;
 pub use text::*;
+
+/// The plugin for UI widgets
+#[derive(Default)]
+pub struct WidgetPlugin;
+
+impl Plugin for WidgetPlugin {
+	fn build(&self, app: &mut App) {
+		app.add_plugin(ButtonPlugin)
+			.add_plugin(TextPlugin)
```

Comment:
There seems to be a mix between tabs and spaces in your code. Make sure your indentation settings are correct, and I'm pretty sure the rest of Bevy is using 4 spaces for indentation.

Reply:
I believe this should have been handled by rustfmt, I'm a bit surprised CI passed on this